Classes/YLTableView.m

```diff
@@ -71,7 +73,7 @@ - (void)registerClass:(Class)headerFooterViewClass forHeaderFooterViewReuseIdent
   }
 }
 
-- (YLTableViewCell *)sizingCellForReuseIdentifier:(NSString *)reuseIdentifier {
+- (nonnull YLTableViewCell *)sizingCellForReuseIdentifier:(nonnull NSString *)reuseIdentifier {
   NSAssert(reuseIdentifier != nil, @"Must have a reuse identifier.");
```

Comment:
1. not specifically in NSAssert but in other places like 
```obj-c
if (refreshHeaderView) {
```
2. Let's keep the `NSAssert` then